setup.cfg

```diff
@@ -1,147 +0,0 @@
-long_description_content_type = text/x-rst
-author = ctapipe developers
-author_email = karl.kosack@cea.fr
-license = BSD 3-Clause License
-url = https://github.com/cta-observatory/ctapipe
-classifiers =
-    Intended Audience :: Science/Research
-    License :: OSI Approved :: BSD License
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
-    Programming Language :: Python :: Implementation :: CPython
-    Topic :: Scientific/Engineering :: Astronomy
-    Development Status :: 3 - Alpha
-
-
-[options]
-packages = find:
-package_dir =
-    =src
-python_requires = >=3.9
-zip_safe = False
-install_requires=
-    astropy >=5.3,<7.0.0a0
-    bokeh ~=2.0
-    docutils
-    eventio >=1.9.1, <2.0.0a0
-    iminuit >=2
-    importlib_metadata  ; python_version < "3.10"
-    joblib
-    matplotlib ~=3.0
-    numba >=0.56
-    numpy ~=1.16
-    psutil
-    pyyaml >=5.1
-    requests
-    scikit-learn <1.4
-    scipy ~=1.2
-    tables ~=3.4
-    tqdm >=4.32
-    traitlets ~=5.6
-    zstandard
-
-
-[options.extras_require]
-tests =
-    pytest >= 7.0
-    pytest-cov
-    pytest-xdist
-    pandas
-    tomli
-    pytest_astropy_header
-    h5py
-
-
-docs =
-    sphinx
-    pydata_sphinx_theme
-    sphinx_automodapi
-    nbsphinx
-    numpydoc
-    sphinx-design
-    sphinx-gallery
-    jupyter
-    notebook
-    graphviz
-    pandas
-    ipython
-    ffmpeg-python
-    pypandoc
-
-dev =
-    setuptools_scm[toml]
-
-all =
-    %(tests)s
-    %(docs)s
-    %(dev)s
-
-[options.packages.find]
-where = src
-exclude =
-    ctapipe._dev_version
-
-[options.package_data]
-* = resources/*
-
-
-[options.entry_points]
-console_scripts =
-    ctapipe-info = ctapipe.tools.info:main
-    ctapipe-dump-instrument=ctapipe.tools.dump_instrument:main
-    ctapipe-display-dl1 = ctapipe.tools.display_dl1:main
-    ctapipe-process = ctapipe.tools.process:main
-    ctapipe-merge = ctapipe.tools.merge:main
-    ctapipe-fileinfo = ctapipe.tools.fileinfo:main
-    ctapipe-quickstart = ctapipe.tools.quickstart:main
-    ctapipe-train-energy-regressor = ctapipe.tools.train_energy_regressor:main
-    ctapipe-train-particle-classifier = ctapipe.tools.train_particle_classifier:main
-    ctapipe-train-disp-reconstructor = ctapipe.tools.train_disp_reconstructor:main
-    ctapipe-apply-models = ctapipe.tools.apply_models:main
-
-ctapipe_io =
-    HDF5EventSource = ctapipe.io.hdf5eventsource:HDF5EventSource
-    SimTelEventSource = ctapipe.io.simteleventsource:SimTelEventSource
-
-
-[build_sphinx]
-source_dir = docs
-build_dir = docs/_build
-all_files = 1
-
-
-[tool:pytest]
-minversion=3.0
-norecursedirs=build docs/_build
-addopts = -v
-astropy_header = true
-filterwarnings =
-    ignore:`np.MachAr` is deprecated:DeprecationWarning
-
-
-[flake8]
-exclude=
-  build,
-  docs,
-  .eggs
-max-line-length=90
-select = C,E,F,W,B,B950
-ignore = E501,W503,E203,W201,E712,E731
```

Comment:
flake8 does not support being configured via pyproject.toml (the maintainer hates it for some reason) and since we intend to move to ruff (#2416) I didn't bother adding it to a .flake8 file